tests/unit/matrix/test_subtotals.py

```diff
@@ -580,98 +588,75 @@ def it_can_compute_a_subtotal_intersection_with_subtrahends(
             subtrahend_idxs=np.array(col_subtrahends),
         )
 
-        subtotals = ZscoreSubtotals(None, None, None)
+        subtotals = ZscoreSubtotals(_base_weighted_counts_, None)
```

Comment:
just `weighted_counts_`, see below comment on fixture. same throughout this test class.